tests/Auth/bot-authentication/Startup.cs

```diff
@@ -33,7 +33,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<MainDialog>();
 
             // Create the bot as a transient. In this case the ASP Controller is expecting an IBot.
-            services.AddTransient<IBot, AuthBot<MainDialog>>();
+            //services.AddTransient<IBot, AuthBot<MainDialog>>();
```

Comment:
Do we need to keep this here?

Reply:
This is just the test project - this is just a tool in the solution - we need something to run with the debugger.

Reply:
I switch to Echo bot for debugging streaming and main for debugging the oauth prompt. So having both lines in there is handy.